src/components/NavBar/SearchBar.tsx

```diff
@@ -312,8 +341,8 @@ export const SearchBar = () => {
           </Box>
           <Box
             as="input"
-            placeholder="Search tokens"
-            width={isOpen ? 'full' : '120'}
+            placeholder={`Search tokens${phase1Flag === Phase1Variant.Enabled ? ' and NFT collections' : ''}`}
```

Comment:
rendered string values should be translated

Reply:
localization is not in the scope of this work and will be a follow up PR

Reply:
We've always enforced localization at the same time as we've written the UI, but I'm not personally opposed to changing that if there is a card already assigned/prioritized to go out with this release. Does that exist somewhere?